src/components/dialogs/create-filter-dialog.js

```diff
@@ -185,6 +187,14 @@ const CreateFilterDialog = ({
         if (!filterNameValid || loadingCheckFilterName) {
             return;
         }
+
+        if (newListType !== FilterType.AUTOMATIC) {
+            setFilterCreationType(newListType);
+            setFilterName(newNameList);
+            handleClose();
+            return;
```

Comment:
Is this to avoid creating a new filter when changing only filter Type or Name ? Seems strange...

Reply:
Maybe CreateFilter should have another argument to edit an existing Filter instead of creating another one ? (PUT instead of POST) ?

Reply:
outdated comment it's ok now